src/worker-runner.js

```diff
@@ -86,11 +86,6 @@ export default async function run({
   await runTestBlock(tests, hasFocusedTests, testNamePatternRE, results, stats);
   stats.end = performance.now();
 
-  snapshotState._inlineSnapshots.forEach(({ frame }) => {
-    // When using native ESM, errors have a URL location.
-    // Jest expects paths.
-    frame.file = fileURLToPath(frame.file);
```

Comment:
If by any chance `frame.file` could be a URL, we can test if `frame.file` starts with `file://` and then apply the transform.

Reply:
https://github.com/facebook/jest/pull/12277, so `27.5` can be minimum if you don't wanna leave 27 behind 🙂

Reply:
Thank you!